src/points/slice.ts

```diff
@@ -39,9 +39,12 @@ export interface State {
   pendingPointsEvents: PendingPointsEvent[]
   pointsBalanceStatus: 'idle' | 'loading' | 'error' | 'success'
   pointsBalance: string
+  trackOnceActivities: {
```

Comment:
nit: i think this name is totally fine as-is, but did you consider e.g. `binaryActivities`? might be a bit more clear, since track _could_ be called for one of these activities multiple times if the first call fails

Reply:
i thought about this for a while (naming this thing has been really hard) but i think i prefer to stick to the current name because `binaryActivities` could read like an object that describes which activities are binary rather than whether we've tracked an activity that should be tracked only once. i hear the point about how these `trackOnce` activities could be tracked multiple times but i guess our intention is to prevent that as much as we can...excluding the reinstall use case, we only call the track endpoint once for these activities (and only multiple times if the track fails, but you could argue that a failed track doesn't count as a track)